content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -194,13 +194,11 @@ The detailed documentation of `kubectl autoscale` can be found [here](/docs/refe
 
 ## Autoscaling during rolling update
 
-Currently in Kubernetes, it is possible to perform a [rolling update](/docs/tasks/run-application/rolling-update-replication-controller/) by managing replication controllers directly,
-or by using the deployment object, which manages the underlying replica sets for you.
-Horizontal Pod Autoscaler only supports the latter approach: the Horizontal Pod Autoscaler is bound to the deployment object,
-it sets the size for the deployment object, and the deployment is responsible for setting sizes of underlying replica sets.
+The Horizontal Pod Autoscaler is bound to the deployment object, it sets the size for
```

Comment:
HPA can target a Deployment, ReplicaSet, StatefulSet, and ReplicationController. 
It should be fixed in a separate PR, though. 
cc owners of this doc - @fgrzadkowski @jszczepkowski @directxman12